homeassistant/components/spokestack/config_flow.py

```diff
@@ -0,0 +1,59 @@
+"""ConfigFlow for Spokestack."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+
+from .const import (
+    CONF_KEY_ID,
+    CONF_KEY_SECRET,
+    CONF_LANG,
+    CONF_MODE,
+    CONF_PROFILE,
+    CONF_VOICE,
+    DEFAULT_LANG,
+    DEFAULT_MODE,
+    DEFAULT_PROFILE,
+    DEFAULT_VOICE,
+    DOMAIN,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(
+            CONF_KEY_ID,
+            msg="key_id",
```

Comment:
the default message talks about missing keys. Your substitute message just mentions the key. BEtter to drop it.
```suggestion
```